components/measurement/useTestKeyController.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useCallback } from 'react'
+import { Flex } from 'ooni-components'
+
+// Custom hook to render a toolbar to enable/disable parts of `testKeys`
+// Usage: Add this to the MeasurementContainer
```

Comment:
Again, I'd like to praise this correct usage of comments to explain the purpose of commented out code

Reply:
I'd perhaps just rephrase the comment to say "This file ..." such that it's clear what is the purpose of this very useful piece of code, which otherwise may not be fully obvious at a first glance.

Reply:
Done. Another way to do this would be to use the `NODE_ENV === 'development'` flag to make it available in dev mode. But this isn't that frequently used either.